spec/factories/state_file_nj_intakes.rb

```diff
@@ -159,6 +159,10 @@
       end
     end
 
+    trait :with_1099_rs_synced do
+      after(:create, &:synchronize_df_1099_rs_to_database)
+    end
+
```

Comment:
[pebble] - i see this in the AZ and MD calculator tests, but not in the NJ ones... do we really need this? Or should we be using it in our calculator tests too?

Reply:
good catch!! this is from debugging the database setup issue, was one of the things I tried (but ultimately unnecessary) and forgot to clean up. thanks